kohese-portal/angular.json

```diff
@@ -47,6 +47,7 @@
             ],
             "scripts": [
               "./build/client/cache-worker/cache-bundle.js",
+              "node_modules/marked/lib/marked.js",
```

Comment:
Why is marked being added to the scripts?

Reply:
[ngx-markdown on npm](https://www.npmjs.com/package/ngx-markdown) has this as step 2 of its installation. The relevant excerpt is here:

As the library is using Marked parser you will need to add node_modules/marked/lib/marked.js to your application.
If you are using Angular CLI you can follow the angular.json example below...
```
"scripts": [
+ "node_modules/marked/lib/marked.js"
]
```